scss/calorie-counter-responsive.scss

```diff
@@ -0,0 +1,31 @@
+@import './mixins';
+
+@include md {
+  body {
+    font-size: 20px;
```

Comment:
It's not a good practice to change the global font size of the body. Try adjusting the styles of your component only